### DISCUSSION TITLE
Luts Preview

### DISCUSSION THREAD

**Discussion by MightyJax** [videovillage/community#104] [Ideas]
SO when I select a LUT on my computer in a folder among others it doesn't take me back to that same spot or give you a preview when you hover over it. Is it supposed to do that or would this be a "feature request".  I

**Comment by gregcotten** [comment]
Hi! This is definitely something Screen doesn't do yet - the menu thing is tricky because macOS doesn't allow you to do this with their native menu system. For previewing a LUT, that isn't a feature we ever considered - can you explain a bit further what you mean?

**Comment by MightyJax** [comment]
Hey Greg,

So what I mean by preview a LUT is there is no way to store your LUTS within the SCREEN itself because of limitations in macOS a specific way,  so when I select a LUT it's from my folder and it's organized by specifics so I know what to look for.  If I hold my cursor over the LUT/File it would be amazing if it can generate a "preview" as if I clicked it, so I can scroll through my options. For me it helps with workflow. If you think of Lightroom you can have Presets that do the same thing. So over all maybe it's allowing SCREEN to access your LUT folder on your computer cache generate previews so you can hover and see it. 
You also could go a step further and have a drawer you can turn on/off that would show said generated cached preview i.e. a snapshot/still with LUT Applied.

**Comment by ghost** [comment]
That would come in very handy! Especially if you would like to present different looks to a client on the fly...

**Comment by MightyJax** [comment]
I could name many other little features but it's unlikely they will implement. Most of these companies have no vision outside their workspace and rely on input to enhance, instead of  Understanding workflow of different film makers and seeing it from a bigger perspective. It unfortunately is what it is.

**Comment by gregcotten** [comment]
Not even true by a longshot.

**Comment by ghost** [comment]
Let's stay kind. No one can solve problems or wishes within seconds.

**Comment by ghost** [comment]
I think it would be great to get a preview of the lut within the video window while you scroll the list of Luts so you get an idea of what would happen if you would select the Lut. In addition it would be great if the selected lut would be like printed in bold. That way it would be way more easy to find the point where you chose your lut within the list.
